README.md

```diff
@@ -116,3 +116,65 @@ By default, 10 entries are returned from spot 0.
 # Return 25 entries starting from spot 32
 DummyElasticSearchModel.where(_size: 25, _from: 32)
 ```
+
+#### Querying with Inheritance
+When querying on a super class of an ElasticSearch Model, it will retrieve all matching documents of that class and all sub classes of the super class.
+```ruby
+# Using the DummyElasticSearchModel from above
+class DummySubClassModel < DummyElasticSearchModel; end
+
+a = DummyElasticSearchModel.create!(my_string: "Hello World", my_other_string: "Foobar")
+b = DummySubClassModel.create!(my_string: "Hello World", my_other_string: "Baz")
+c = DummySubClassModel.create!(my_string: "Goodbye")
+
+DummyElasticSearchModel.where
+=> Returns [a, b, c]
+
+DummyElasticSearchModel.where(my_string: "Hello World")
+=> Returns [a, b]
+
+DummyElasticSearchModel.where(my_other_string: "Baz")
+=> Returns [b]
+
+DummySubClassModel.where
+=> Returns [b, c]
+
+DummySubClassModel.where(my_other_string: "Foobar")
+=> Returns []
+``` 
+
+#### Querying across multiple indices
+To query across multiple indices, add the `_indices` parameter in your query with an array of the indices you wish to query across.
+Not specifying the indices in a query will use the index name returned from `index_name` by the model you are querying.
```

Comment:
I'd suggest updating this section and the examples to note that the intended usage for this is to be able to query multiple indices for 1 class (or at least 1 super class) and that this is mainly to support setups where indices are sharded by time.
```
class DummyDailyIndexModel < DummyElasticSearchModel
  def index_name
    time.strftime("dummy_daily_index.%Y.%m.%d")
  end
end
```
```
DummyDailyIndexModel.where(_indices: ["dummy_daily_index.2019.01.05", "dummy_daily_index.2019.01.06", "dummy_daily_index.2019.01.07"])
```

You could leave a comment that the `_indices` names given are arbitrary and so it is possible to search in any index, but that you may not get back any models since we query for matching class type(s).